general/adios2stream.cpp

```diff
@@ -0,0 +1,747 @@
+/*
+ * Distributed under the OSI-approved Apache License, Version 2.0.  See
+ * accompanying file Copyright.txt for details.
+ *
+ * adios2stream.cpp : implementation of adios2stream functions
+ *
+ *  Created on: Feb 4, 2019
+ *      Author: William F Godoy godoywf@ornl.gov
+ */
+
```

Comment:
Merge master in this branch and replace the above with the new BSD license.

If this really needs to be under a different license, then mention it in the top-level `NOTICES` file

Reply:
Done, no need for a different license, my bad. Thanks!